test/DGmethods/compressible_Navier_Stokes/rising_bubble-model.jl

```diff
@@ -110,7 +103,9 @@ function run(mpicomm, ArrayType,
                      SmagorinskyLilly{DF}(C_smag),
                      EquilMoist(), 
                      NoRadiation(),
-                     source_geopot!, NoFluxBC(), initialise_rising_bubble!)
+                     Gravity(),
```

Comment:
Shouldn't this by a tuple? (e.g., `(Gravity(), )`)

Reply:
The idea is that you can specify either a single source, or a tuple of sources.